validator/src/main.rs

```diff
@@ -1421,6 +1425,12 @@ pub fn main() {
                 .conflicts_with("no_accounts_db_caching")
                 .hidden(true)
         )
+        .arg(
+            Arg::with_name("disable_duplicate_instance_check")
+                .long("disable-duplicate-instance-check")
+                .takes_value(false)
+                .help("Disables duplicate instance check"),
```

Comment:
I think we'll want to `.hidden(true)` this one

Reply:
good point, done